cli/dts/lib.deno.ns.d.ts

```diff
@@ -114,6 +114,139 @@ declare namespace Deno {
    * See: https://no-color.org/ */
   export const noColor: boolean;
 
+  export type PermissionOptions = "inherit" | "none" | {
```

Comment:
The export is "unnecessary" but consistent.

I would consider doing something like this, as it will make the documentation on doc.deno.land better:

```suggestion
  export type PermissionOptions = "inherit" | "none" | PermissionOptionsObject;
  
  export interface PermissionOptionsObject {
```